console.py

```diff
@@ -160,5 +160,18 @@ def do_update(self, arg):
         else:
             print("** class doesn't exist **")
 
+    def do_count(self, args):
+        """Count current number of class instances"""
+        count = 0
+        __objects = storage.all()
+        for k, v in __objects.items():
+            if args == k.split('.')[0]:
+                count += 1
+        print(count)
+
+    def help_count(self):
+        """ """
```

Comment:
Shouldn't leave docstrings blank. Could be something as simple as "prints correct usage if count is used wrong".

Reply:
Good catch!